src/server/network/network.ts

```diff
@@ -85,56 +93,86 @@ export class UDPServer {
     return () => this.clients.delete(client);
   }
 
-  // Set up callbacks for a given team and then bind the team to its multicast address and port
-  private initialiseListener = (listener: Listener) => {
-    listener.server.on('listening', () => {
+  // One of the UDP servers received a new message, send it on to all clients
+  private onMessage = (
+    team: TeamData,
+    packet: Buffer,
+    rinfo: dgram.RemoteInfo,
+  ) => {
```

Comment:
Is there a reason that you made a variable equal to a function rather than a member function? There are some reasons to do that but I think in this case it should just be a normal function
```suggestion
  private onMessage(
    team: TeamData,
    packet: Buffer,
    rinfo: dgram.RemoteInfo,
  ) {
```